src/main/java/org/dependencytrack/model/Severity.java

```diff
@@ -47,4 +47,26 @@ public int getLevel() {
     public static Severity getSeverityByLevel(final int level){
         return Arrays.stream(values()).filter(value -> value.level == level).findFirst().orElse(UNASSIGNED);
     }
+
+    public String getSeverityAsString(){
+        if(level == 0){
+            return "UNASSIGNED";
+        }
+        if(level == 1){
+            return "INFO";
+        }
+        if(level == 2){
+            return "LOW";
+        }
+        if(level == 3){
+            return "MEDIUM";
+        }
+        if(level == 4){
+            return "HIGH";
+        }
+        if(level == 5){
+            return "CRITICAL";
+        }
+        return null;
+    }
```

Comment:
JAVA has built in syntax for this:
Severity s = Severity.MEDIUM; 
System.out.println(s.name()); <<- This returns "MEDIUM"